pkg/networkserver/grpc_gsns.go

```diff
@@ -713,10 +715,10 @@ func (ns *NetworkServer) handleUplink(ctx context.Context, up *ttnpb.UplinkMessa
 
 // newDevAddr generates a DevAddr for specified EndDevice.
 func (ns *NetworkServer) newDevAddr(context.Context, *ttnpb.EndDevice) types.DevAddr {
-	nwkAddr := make([]byte, types.NwkAddrLength(ns.NetID))
+	nwkAddr := make([]byte, types.NwkAddrLength(ns.netID))
```

Comment:
Please file and issue and TODO here that we generate `DevAddr` from a `DevAddrPrefix`, which falls back to the `NetID`. But in clustered NS, we're using a subset of the `NetID`.